src/devices_models/devices/DC_branches.jl

```diff
@@ -14,9 +14,9 @@ flow_variables!(
     devices::IS.FlattenIteratorWrapper{<:PSY.DCBranch},
 ) = nothing
 
-function flow_variables!(
+function add_variables!(
     psi_container::PSIContainer,
-    system_formulation::Type{StandardPTDFModel},
+    system_formulation::StandardPTDFModel,
```

Comment:
The system formulation variable is not used here. It can be removed.

Reply:
Resolved in 40e2de21.

Reply:
Resolved in 461c2777 instead.